### ISSUE TITLE
Inclusion of MASE diagnostics

### ISSUE THREAD

**Issue by alkuzad** [facebook/prophet#1183]
I am experimenting with forecasting data that is very seasonal, weekend have often measurements of 0 due to nature of 
the dataset. This naturally crosses out MAPE.

Also I found out that using MAE is problematic as each data series is in different scale (some datasets caps around 100, some goes into thousands) and for some MAE of 4 is too much, some cross 30.

I was looking for some other metrics to validate if our forecasting model is better than just guessing that data should be same as in previous week.

Do you see any potential problems implementing this method in diagnostics for Prophet ? 

I'm trying to implement it on example data and it could be beneficial to include it as standard, this would need calculating seasonal naive metric and decide which seasonality it should use, I do not know if this can be decided automatically or just avg/worst result of all of them could be present.

**Comment by bletham** [comment]
I think that makes a lot of sense as a benchmark. Basically it would involve computing MAE on cross validation sets for both PRophet and SNE, and then you could compare those numbers to determine if Prophet is doing a reasonable job. This is something that we've done (and in fact is something that we recommend in Section 4.1 of the paper, https://peerj.com/preprints/3190.pdf). As noted there, you might even want to compare to a few baseline methods.

I think it would be challenging to integrate other methods into the cross_validation tool inside the package. I think the best approach would be to use the package's cross_validation to compute MAE for Prophet, and then externally do cross validation for the other model(s) to compute their MAE. This could be made easier by having a simple way to export from Prophet the train/test dates used in the cross validation so we can be sure those are the same - that would be a doable change.
